test/grpc/route_guide/route-guide-eventuals-client.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <grpcpp/channel.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "eventuals/grpc/client.h"
+#include "test/grpc/route_guide/helper.h"
+#include "test/grpc/route_guide/route_guide.grpc.pb.h"
+
+using eventuals::grpc::Client;
+using eventuals::grpc::CompletionPool;
+using routeguide::Feature;
+using routeguide::Point;
+using stout::Borrowable;
+
+////////////////////////////////////////////////////////////////////////
+
+class RouteGuideBaseClient {
```

Comment:
Was compiling really slow and that's why you did this? I want to be as close as possible to the reference implementation so let's put the entire client in a single file as long as it is not really slow to compile. And if it is, can we declare each of the functions as returning `auto` and then only define the functions in `.cc` files where we also have the tests?